### ISSUE TITLE
pictures from de.serlo in en.serlo don't work

### ISSUE THREAD

**Issue by SimonKoehl** [serlo-org/athene2#159]
It's not possible to include pictures which are uploaded in de.serlo in en.serlo with [](/attachement/...). However it works with the complete link [](https://de.serlo.org/atachement/...) Is that for a reason? Can/Should that be fixed?

**Comment by inyono** [comment]
Seems like the attachment don't resolve the url tenant-independent. See error log when using the relative url in the wrong tenant (encoded since I can't remember the url for the decoder):
```
Q/Pr6JRb4JLHmSekWHeWq8dqotBzBJli6nqgiNGmLDDLwuSfYmunUI36aPUHbva4Up9alKwRqdun1VMaHbaSZGpMEF3ASETHkMXSb8wu1DT93J/JzIR/3CgPZ6oQubreVd1rh/lytbyy2YWslxukTqtBFrwiOGkug0KDALCoiZ9Bn/SNDNxVns8sqOK4o91TiAt8ntk0vuWvEESzA7HUiq7gm1kAuurbB9zAEol47+rmANzfiu9/Ko2WYRAdn9IWRSNZi7odnjlf9DHGP5DUSMhOyLRblTqQ1fYaWMl8tVGM51XjDUYZiyy4JWl0Iuy9PwtSldJiA8KRo0OgfK+X6yXwRyxVit0/ha8bEPLlykuASuAEM1ljnjtBFQN8F4USPp6M9Rnz9yYnXr8i96sQzDbsc8X7dcEHlLtPVzBzLgYuDehK8kfW4HYPQCzw4hK1dIZIX06mPEPqhBlau11CRUcw+M9M87YhniayJ/stFnuVFD5AyzEyWRXA9sXRZzqJP52qDsemGb5qHR5OWC1UVLyvW1UqkWUutCGXwdfav4Lp90cKclDRpi9KPEDGFoxAkp+LS81s9RGpCU4peni5fZ/c8pczLFXX+47KX/sseAfpZxZJd9iR8l3obPuUXGH3RLOBzDYiub4fCOJvrPfEZS8T4DEX1G5q3y1CyeTEKvmSGiIdrcfwE9zXXU3eUxq5qG32pO/mriceqQLGVQU+NhO/rwst6GUV3iqGS0QtKVsbROqmWA6+nFX9GOPXiQ4cjT1lC1P9bgwsHlClDV71edM+8CNCJpBgzIuYqZQ3gFtf8LAmD1NfKL9SRA/nltrGPbp8AkbFMnFOeHRYUI85SUsFqDzOy/uN2NPubrz6z1JKsGHSkESQjXaYfFnf1VWjWOVDTYSjZC4gKOjojj3URGMsvFqTOrZ2iL1kGCWsrCtfS9GTaLgfU/umnB94p900U9DFOUvWYoCluNtyklAsmCV8MG91/K9Y53r6rg3E6zfJdd0GgPXy8wnzi2BNDzKxL6HQfberRSsl2JjvNW5Rkf+DweTS5AZ2PQhTx5FYrERdvAUMgKojvIvOVMOWkAPl9mYko7J94DEgjImDoreh+cc4k8Yn++PgTW8z/CLyQkNMYy3CZAhNOCqirXAJnnXla7qw7UG0Chrz6hYfhws+FQ8bXEiQrvyX53Es3v+fulxc6viVYB/N70f5Io0IpKw8fAdyxezw9kWAw/JiK8hOY2FwiLa39LYKYsR6LJbi8MfVb3WzxfuanClxn6iGhv/vWDYvk6eYqDa7zkAcwQVuZte8h7f38LcvOpEaKC/WUrLl2BdxLx+VMLthIo0w5UlEFWREMerizXf/Ii9ibIJULobUbWo9hfUIK0j1vmIspIk1vwDFqQPibswDbFlP+Yq022OJjrSWHpiGdbWWZwYsnOLKypsxC87Rf2ncRy5TgNFIAPYZ6hc0vfOCkpF0unn/6kaFZ8pr07ofDlRRR+UgWnIF8gmpFrYVozSD6r4vTp1xXH1jcbkn2JkIsM5P2vM2jlbVYQGdfFU1wkSa4pAU4rCxO+qtOAOqVB3apIIg9Cf3QZuE4dx2dsV1NqB/JOMSqw5byqMCH35plC0Q/1KbXDRBjsi1PQ+4QLFyiYa7jepeSV6nadVCCT6RFbnwTrq49GmwkXYasPJ7evwWvcyFXkIRsTQdAWdeBFaaHr1jt2e3kj2fjRQRteiT1+FE/8UKOxhEGAZiJfgkNsIuH82ieinLJgAtLEBnIHyW6FalokoulZeirg2udclYK9gEsxyfZJ1e4om3PaA9wz7nwzW8wF6fAvtp7eRr/VexqD2fnBOByVJRPP9rkrodzTsbPWbHcRTBVEncZXC8baLc+MhQqF1+gOrAzl/Ci0LMjTkZ30VOeoTz3V4n+WN3oeXi1pC/T7b99Vf3TKgBuSqE9RbvcRfeyyol+0yR3h8EdxfkS+NO6zXtWqL94DOA+EkiI8+fHbgEfM7IGkSmJqMa0WDV/vHdb3U4syWOEKZ/6U8JzDw3hK+uL0rhixIXIGLFX1coDJytc3n2yJULbN38u1WF5+Qu1GuZgk6CjcmunGZ5UYjoOALw0E1dkQdsvrGuqwXQGaxr2H8Cn2iJRrGcHtvCEXEoAe8H4P7MP2L3QC9Xv3whndbmMT72tzBA1XKH6cd7o6NupMX9n796LzrfyvONLGav8fPdz91z8TmvpYSTbtzkRtOtQKpPF2t3VWwul7n/+dEoe7sDBKimdQ/hf1smtZxOJUGs1WlEgKCJj8XUfDjq5wyGJASlAKOrkQbkWsX7PwdFaHPru8LmoXxGx2htfyhrU1FQ914mLFb7JJAenK3Ng8S+KXUXjRHPQSOInsfzRqNHle7xI3yZg7n5sb3rWpg5oqjqvVSnp79uZljT2ju/+whZK8pN/aO5HhRqRWSZ8lsAaCJduTWgnVDNtHNYqRHWMpi1sRKvk2/MMjo75SvDw5gxhgVlC8ba4aCblmAxQDJ3W43XcdoIHIh5VQCEnVmZIrRL+ETH80SZF3PS1xn11xvDyoL6rc9lfLKhzHcsRdhOFmKTWk6HSWq5UKPHhepyJWoTmR2NS22tBN5+tMuLITh/WHmXjy6T5FRFTaFCYtvpOeZGpFJxI6eLqdL/vKoQwHjl/2TU2cTHCHSXGBVm092wjBBFguR2Z8QEO4yFPtsdbLAOMRDSedAmZIRAXppqcWfbYjW6SAlLt8f2R4qOtg1NnbsGDQgFGHUh5yQgAvKEkb7HduSkLXKwFpX1hFRyZeOM7V7/noFLJkBtEvMU8/2BnbD7/lnaZispXjRhPTmDqW3NmGt7DkAjg9AtPK2ArPqh06ZdZeR7eWHD4VrI4fPXxKCXoTWOU6GDK3JshnJ4D3ggZkad9hoY1POrwqHtd3x7aiaNWbTbeslbKILw6jPh5JVAio59u9NtcMqp7AhvhqDNACsgk9i1aF/oRpM0cljA+8vuc5xhCjdNYf1J00MqUW6tYpqf4jj0bCTaKf/DZkMiz6YPaw2FAU92TPNJHT3/RUnDEBxvYeEb9fUnrLnfjVZklXV8RdIl6d4AITyvFNxXc8xMfv4xTuXiadLaVuvVidknyq14Cjdj1+eE447swdpNvvh3eGfF+L9qaSpNsZJ84VWCBdL8BpQnURHCA2cCn99kCDKrBbWWWBq7igBmt44IeFFLx/RNU8o5xXmVdRUEdIn51iggoLry83y0n0GkTrf+/J0QXUXqsWCJNdz3KsVsve9MO4tkEFt8R5KT82jxxtYMsbSUs/I/mA==
```
I'd classify that as a bug, since the attachment's id should be unique as everything else...

**Comment by arekkas** [comment]
Since reuploading pictures works fine, I'd classify this as low priority

**Comment by SimonKoehl** [comment]
https://en.serlo.org/user/profile/34442

not urgend: reuploading is not an option on profile pages. (But anyway we have to think through multi-tenat profile handling separately )

**Comment by Knorrke** [comment]
this should be solved by #530 for new uploads. If you have a link like `/attachment/file/.../...` anywhere please open the url and replace the link with the redirected `https://assets.serlo.org/....` link now showing as url. That link works on de.serlo as well as en.serlo
